Kitodo-API/src/main/java/org/kitodo/config/OPACConfig.java

```diff
@@ -43,8 +44,18 @@ private OPACConfig() {
      * @param parameter String identifying the parameter by name
      * @return value of parameter
      */
-    public static String getConfigValue(String catalogName, String parameter) {
-        return getCatalog(catalogName).getString(parameter);
+    public static String getConfigValue(String catalogName, String parameter) throws ParameterNotFoundException {
+        HierarchicalConfiguration opacConfiguration = getCatalog(catalogName);
+        if (Objects.isNull(opacConfiguration)) {
+            throw new ParameterNotFoundException("No configuration foudn for catalog '" + catalogName + "'!");
+        } else {
+            String parameterValue = opacConfiguration.getString(parameter);
+            if (parameterValue.isEmpty()) {
```

Comment:
The use of StringUtils.isBlank(parameterValue) is better in this case because getString() method can return a null.